product_packaging_container_deposit/models/container_deposit_order_mixin.py

```diff
@@ -86,7 +86,9 @@ def update_order_container_deposit_quantity(self):
                         product, deposit_container_qties[product][0]
                     )
                     values_lst.append(Command.create(values))
-            order.write({self._get_order_line_field(): values_lst})
+            order.with_context(update_order_container_deposit_quantity=True).write(
```

Comment:
there's already the key `skip_update_container_deposit` at line 36: do you really need this?

Reply:
not exactly the same purpose:
- skip_update_container_deposit: avoid update on copy()
- update_order_container_deposit_quantity: flag to circumvent the fields protection

Reply:
Thank @nilshamerlinck and @simahawk , I updated